tests/integration/test_postgres.py

```diff
@@ -180,6 +181,38 @@ def test_get_jobs(job_store):
     ]
 
 
+def test_get_stalled_jobs(get_all, job_store):
+    job_store.register_queue("queue_a")
+    job_store.launch_job(
+        jobs.Job(
+            id=0,
+            queue="queue_a",
+            task_name="task_1",
+            lock="lock_1",
+            task_kwargs={"a": "b"},
+            job_store=job_store,
+        )
+    )
+
+    # No started job
+    assert list(job_store.get_stalled_jobs(nb_seconds=3600)) == []
+
+    # We start a job and wait a little
+    job = next(job_store.get_jobs("queue_a"))
+    time.sleep(1)
```

Comment:
Not particularly fond of this :O Since we are using psql `NOW` I don't see an easy way to cheat our way out of this :\

Reply:
`UPDATE cabbage_task SET started_at='2000-01-01T00:00:00+00:00'::timestamp with timezone WHERE id = %(id)s` ?

Reply:
J'ai fait quelque chose comme ca, merci :+1: